google/cloud/forseti/scanner/audit/kms_rules_engine.py

```diff
@@ -346,34 +338,107 @@ def __init__(self, rule_name, rule_index, rule):
         self.rule_index = rule_index
         self.rule = rule
 
-    def is_key_rotated(self, creation_time, scan_time):
-        """Check if the key has been rotated within the time speciifed in the
-         policy.
+    def find_match_rotation_period(self, key, rotation_period, mode):
```

Comment:
Using match in the method name is unclear in this case, when comparing rotation_period, it's looking for either greater than/equals to or less than/equals to and not looking for exact match.

Reply:
It's looking if match is found or not based on the mode and `days_since_rotated`. We want to throw violations only if `days_since_rotated  > rotation_period` and the method is doing exactly that.